restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/MediaClient/AppRTCAudioManager.java

```diff
@@ -403,6 +373,38 @@ private void registerForWiredHeadsetIntentBroadcast()
          @Override
          public void onReceive(Context context, Intent intent)
          {
+
+            // Create and initialize the proximity sensor.
+            // Tablet devices (e.g. Nexus 7) does not support proximity sensors.
+            // Note that, the sensor will not be active until start() has been called.
```

Comment:
General issue: When testing these changes I'm getting a crash when I start call, plug in and then plug out the wired headset